addons/cargo/functions/fnc_addCargoItem.sqf

```diff
@@ -7,31 +7,40 @@
  * 0: Item to be loaded <STRING> or <OBJECT>
  * 1: Holder object (vehicle) <OBJECT>
  * 2: Amount <NUMBER> (default: 1)
+ * 3: Ignore interaction distance and stability checks <BOOL> (default: false)
  *
  * Return Value:
- * None
+ * Objects loaded <NUMBER>
  *
  * Example:
  * ["ACE_Wheel", cursorObject] call ace_cargo_fnc_addCargoItem
  *
  * Public: No
  */
 
-params ["_item", "_vehicle", ["_amount", 1]];
-TRACE_3("params",_item,_vehicle,_amount);
+params ["_item", "_vehicle", ["_amount", 1], ["_ignoreInteraction", false]];
+TRACE_4("params",_item,_vehicle,_amount,_ignoreInteraction);
+
+private _loaded = 0;
 
 // Get config sensitive case name
 if (_item isEqualType "") then {
     _item = _item call EFUNC(common,getConfigName);
 
     for "_i" from 1 to _amount do {
-        [_item, _vehicle] call FUNC(loadItem);
+        if !([_item, _vehicle, _ignoreInteraction] call FUNC(loadItem)) exitWith {};
+
+        _loaded = _loaded + 1;
     };
 } else {
-    [_item, _vehicle] call FUNC(loadItem);
+    _loaded = parseNumber ([_item, _vehicle, _ignoreInteraction] call FUNC(loadItem));
 
     _item = typeOf _item;
 };
 
+TRACE_1("loaded",_loaded);
+
 // Invoke listenable event
 ["ace_cargoAdded", [_item, _vehicle, _amount]] call CBA_fnc_globalEvent;
```

Comment:
Do we want to replace `_amount` with `_loaded` or just add `_loaded` to the event?

Reply:
I think so, better to say what actually happened